tutorials/ldbc/0_00_schema.gsql

```diff
@@ -0,0 +1,33 @@
+USE GLOBAL
+
+# Note: outdegree_by_edgetype is turned on for Comment and Person
+CREATE VERTEX Comment (PRIMARY_ID id UINT, creationDate DATETIME, locationIP STRING, browserUsed STRING, content STRING, length UINT) WITH primary_id_as_attribute="TRUE", STATS="outdegree_by_edgetype"
```

Comment:
@YipingXiongTG  can you change primary_id to primary key? and remove "primary_id_as_attribute=true"?